h2o-r/h2o-package/render-h2o.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+### Render h2o R package from h2o-3 as a standalone R package h2o
+# script will download recent nightly h2o.jar so you should always rebase your h2o package to recent master
```

Comment:
Yes, if we have `inst/java/h2o.jar` already in `h2o-package` we can use this. Rebase to latest and re-download jar ensure there are no breaking changes between h2o R package and h2o jar, something like strict version check.